packages/manager/apps/hub/src/components/billing-summary/BillingSummary.component.tsx

```diff
@@ -0,0 +1,244 @@
+);
+
+export default function BillingSummary() {
+  const { t } = useTranslation('hub/billing');
+  const {
+    shell: { navigation },
+    environment,
+  } = useContext(ShellContext);
+  const locale = environment.getUserLocale();
+  const { trackClick } = useOvhTracking();
+  const [months, setMonths] = useState(1);
+
+  const {
+    isPending: areBillsLoading,
+    data: bills,
+    error: billsError,
+    refetch,
+  } = useFetchHubBills(months);
+  const { isPending: isDebtLoading, data: debt } = useFetchHubDebt();
+
+  const isLoading = areBillsLoading || isDebtLoading;
+
+  const getPeriodFilter = ({ from, to }: Period) => [
+    {
+      field: 'date',
+      comparator: 'isAfter',
+      reference: [from],
+    },
+    {
+      field: 'date',
+      comparator: 'isBefore',
+      reference: [to],
+    },
+  ];
+
+  const getFormattedPrice = (price: number, currency: string) =>
+    currency && price
+      ? Intl.NumberFormat(locale.replace('_', '-'), {
+          style: 'currency',
+          currency,
+        }).format(price)
+      : '';
+
+  const formattedPrice = getFormattedPrice(bills?.total, bills?.currency?.code);
+  const formattedDebtPrice = getFormattedPrice(
+    debt?.dueAmount?.value,
+    debt?.dueAmount?.currencyCode,
+  );
+
+  const payDebtLink = useMemo(
+    () => navigation.getURL('dedicated', '#/billing/history/debt/all/pay', {}),
+    [],
+  );
+
+  const viewBillsLink = useMemo(
+    () =>
+      navigation.getURL('dedicated', '#/billing/history', {
+        // From BFF's code, it seems that period cannot be null or undefined, so this code could probably be simplified
+        filter: bills?.period
+          ? JSON.stringify(getPeriodFilter(bills.period))
+          : '',
+      }),
+    [bills?.period],
+  );
+
+  const trackGoToBilling = () => {
+    trackClick({
+      actionType: 'action',
+      actions: ['activity', 'billing', 'show-all'],
+    });
+  };
+
+  return (
+    <div data-testid="billing_summary" className="manager-hub-billing-summary">
+      <OsdsText
+        className="block mb-6"
+        level={ODS_TEXT_LEVEL.subheading}
+        size={ODS_TEXT_SIZE._200}
+        color={ODS_THEME_COLOR_INTENT.text}
+      >
+        {t('hub_billing_summary_title')}
+      </OsdsText>
+      {!isLoading && billsError && (
+        <Suspense
+          fallback={<OsdsSkeleton data-testid="tile_error_skeleton" inline />}
+        >
+          <TileError
+            contrasted
+            message={t('hub_billing_summary_display_bills_error')}
+            refetch={refetch}
+          />
+        </Suspense>
+      )}
+      {!billsError && (
+        <div>
+          <OsdsSelect
+            data-testid="bills_period_selector"
+            className="w-full m-auto max-w-60 px-4"
+            value={months}
+            onOdsValueChange={(event) =>
+              setMonths(Number(event.detail.value as string))
+            }
+          >
+            {[1, 3, 6].map((month) => (
```

Comment:
Wouldn't it be better to add explicit variables, for example "const months = [...]" to indicate what these numbers correspond to?